README.md

```diff
@@ -16,9 +16,9 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 http_archive(
     name = "golink",
-    urls = ["https://github.com/nikunjy/golink/archive/v1.0.0.tar.gz"],
+    urls = ["https://github.com/nikunjy/golink/archive/v1.1.0.tar.gz"],
     sha256 = "ea728cfc9cb6e2ae024e1d5fbff185224592bbd4dad6516f3cc96d5155b69f0d",
```

Comment:
@nikunjy do you know offhand what the new sha256 is?

Reply:
`c505a82b7180d4315bbaf05848e9b7d2683e80f1b16159af51a0ecae6fb2d54d`

tested here https://github.com/nikunjy/go/blob/2caac11b78c14d64da1d6cf908c6b8acd3824019/WORKSPACE#L20

Reply:
thanks!! added another commit